docs/content/Getting Started Using.md

```diff
@@ -119,7 +119,7 @@ dotnet msbuild YourSolution.sln /t:AnalyzeFSharpProject
 
 ### Select specific projects
 
-As we don't want to targt all projects of the solution, we create a second custom MSBuild target that calls the project-specific target for all relevant projects.  
+As we don't want to target all projects of the solution, we create a second custom MSBuild target that calls the project-specific target for all relevant projects.  
```

Comment:
Whilst in the area - would 'in the solution' sound better than 'of the solution'?

Reply:
Yeah, that reads better to my American English ears.